tests/phpunit/api/v3/ContributionTest.php

```diff
@@ -236,7 +239,6 @@ public function testCreateCheckContribution() {
     $params['amount_level'] = 'Unreasonable';
     $params['cancel_reason'] = 'You lose sucker';
     $params['creditnote_id'] = 'sudo rm -rf';
-    $params['tax_amount'] = '1';
```

Comment:
This is removed because the test operates with tax disabled so it is not valid to have a tax_amount for this contribution